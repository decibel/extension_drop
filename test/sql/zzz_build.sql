\set ECHO none
BEGIN;
\i test/pgxntool/psql.sql

CREATE EXTENSION IF NOT EXISTS cat_tools;

\echo
\echo INSTALL
\t
\i sql/extension_drop.sql

\echo # TRANSACTION INTENTIONALLY LEFT OPEN
