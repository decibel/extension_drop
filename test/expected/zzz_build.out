\set ECHO none

INSTALL
 

 

 

 

 

 

 

 

 

 

# TRANSACTION INTENTIONALLY LEFT OPEN
